src/Tooltip/desktop-view.js

```diff
@@ -0,0 +1,222 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ReactMarkdown from 'react-markdown'
+import styled from 'styled-components'
+
+class DesktopView extends Component {
+  state = {
+    hover: false,
+    margin: -70,
```

Comment:
probably not related to this PR even, but it's a really bad practice to hard code constants like this. At the very least there should be constants defined in this file with some explanations where do they come from. Values should be defined as formulas that use those constants. Ideally, certain constant values should be taken from other globals - with of the sidebar, for example, etc.

Reply:
@shcheklein I agree. I will define them as constant and write comments for explanation.

Reply:
Moved conversation here: #663